Javascript(ES6)/README.md

```diff
@@ -3126,9 +3126,8 @@ Forked from the excellent [Airbnb JavaScript Style Guide](https://github.com/air
 **Tools**
 
   - Code Style Linters
-    + [ESlint](http://eslint.org/) - [Airbnb Style .eslintrc](https://github.com/airbnb/javascript/blob/master/linters/.eslintrc)
+    + [ESlint](http://eslint.org/) - [Making Sense Style .eslintrc](.eslintrc)
     + [JSHint](http://jshint.com/) - [Airbnb Style .jshintrc](https://github.com/airbnb/javascript/blob/master/linters/.jshintrc)
```

Comment:
I agree -- also, having more than one tool in the same project is not a very good idea. It is very likely that they'll start conflicting in some particular rule and you need to juggle around your bits to comply with both, or just relax one of the rules "just because".